fsspec/asyn.py

```diff
@@ -297,15 +297,15 @@ class AsyncFileSystem(AbstractFileSystem):
     mirror_sync_methods = True
     disable_throttling = False
 
-    def __init__(self, *args, asynchronous=False, loop=None, batch_size=None, **kwargs):
+    def __init__(self, *args, asynchronous=False, loop=None, batch_size=None, listings_cache_options=None, **kwargs):
```

Comment:
Should we name it `dir_cache_options` instead to adhere with `MemoryDirCache/FileDirCache`?